public/locales/gr/media.json

```diff
@@ -0,0 +1,4 @@
+{
+    "page-title": "On parle de nous",
```

Comment:
Maybe worth using english for all non french / non translated lang?

Reply:
Yes! Done